qiling/debugger/qdb/qdb.py

```diff
@@ -305,17 +303,15 @@ def do_breakpoint(self: QlQdb, address: Optional[int] = 0) -> None:
         print(f"{color.CYAN}[+] Breakpoint at 0x{address:08x}{color.END}")
 
     @parse_int
-    def do_continue(self: QlQdb, address: Optional[int] = 0) -> None:
+    def do_disassemble(self: QlQdb, address: Optional[int] = 0, *args) -> None:
```

Comment:
What is the `*args` for here (and in the following methods)?
A well-defined API usually doesn't need that.